src/pages/references/stacks-blockchain.md

```diff
@@ -1,19 +1,9 @@
 ---
-title: Stacks Blockchain APIs
-description: Interacting with the Stacks 2.0 Blockchain
+title: Stacks Blockchain API
+description: Interacting with the Stacks 2.0 Blockchain via API
 ---
 
-## Introduction
-
-With the launch of Stacks 2.0, a new version of the Blockstack blockchain was released. There are two ways of interacting with the blockchain, either using the Stacks Blockchain API or by making RPC calls to a Stacks Core directly.
-
-## Stacks Core API
-
-The Stacks 2.0 blockchain's Rust implementation exposes RPC endpoints (in JSON format), which can be used to interface with the Stacks blockchain. [You can find the RPC API references here](/references/stacks-rpc-api).
-
-## Stacks Blockchain API
-
-The Stacks Blockchain API was built to maintain pageable materialized views of the Stacks 2.0 Blockchain. It is a server that exposes a RESTful JSON API, hosted by PBC. It introduces aidditonal functionality (e.g. get all transactions), as well as proxies calls directly to Stacks Node. [You can find the OpenAPI specification and documentation here](https://blockstack.github.io/stacks-blockchain-sidecar/).
+The Stacks Blockchain API was built to maintain pageable materialized views of the Stacks 2.0 Blockchain. It is a server that exposes a RESTful JSON API, hosted by PBC. It introduces aidditonal functionality (e.g. get all transactions), as well as proxies calls directly to Stacks Node. [You can find the OpenAPI specification and documentation here](https://blockstack.github.io/stacks-blockchain-api/).
```

Comment:
It's important that we have some initial heading here, whether it be `## Overview` or `## Introduction` so that our content is wrapped correctly in `<section>` elements.

Reply:
Done